### PR TITLE
Introduction to spinalcordtoolbox setup

### PR DESCRIPTION
**Pull request by peristeri**
Spinalcordtoolbox will be added to the python's system path by running

    python setup.py install

or for setting up the development environment by running

    python setup.py develop

This does not change the installation process of spinalcordtoolbox

### CODE REVIEW

**Comment by jcohenadad** [comment]
I successfully ran ```python setup.py develop```. It used my default Python though (and it updated other modules, including scipy, which might be annoying for people). I guess the future plan is NOT to interact with the user's native Python but to use SCT's own Python?

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+with open(path.join(here, './install/requirements/requirementsPip.txt')) as f:
+    requirements = f.read().splitlines()
+
+with open(path.join(here, 'version.txt')) as f:
+    version = f.read().split()
+
+setup(
+    name='spinalcordtoolbox',
+    description='Library of analysis tools for the MRI of the spinal cord',
```
Comment (jcohenadad) [inline setup.py]: Library of analysis tools for MRI of the spinal cord

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        'Topic :: Scientific/Engineering :: Medical Science Apps.',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: Unix',
+        'Operating System :: MacOS',
+        'Operating System :: Microsoft :: Windows',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+    ],
+    keywords='Image MRI spinal cord',
```
Comment (jcohenadad) [inline setup.py]: keywords='Magnetic Resonance Imaging MRI spinal cord analysis template'

**Comment by peristeri** [comment]
The wording of my message might be misleading. I assumed that the user activated the conda python that was installed with SCT. What is planned in the future is the complete integration of SCT in a python environment.

**Pull request merged**